src/utils/env.ts

```diff
@@ -8,28 +8,21 @@ const PC_ENV_TLDS = [
     'usw2.pure.cloud',
     'euw2.pure.cloud',
     'cac1.pure.cloud',
-    'apne2.pure.cloud'
+    'apne2.pure.cloud',
+    ...__PC_DEV_ENVS__
 ];
 
 export interface PcEnv {
     pcEnvTld: string;
     pcAppOrigin: string;
 }
 
-let DEFAULT_PC_ENV!: PcEnv;
-const PC_ENVS = PC_ENV_TLDS.reduce<PcEnv[]>((result, currEnvTld) => {
-    const currEnv: PcEnv = {
-        pcEnvTld: currEnvTld,
-        pcAppOrigin: `https://apps.${currEnvTld}`
-    };
+const PC_ENVS = PC_ENV_TLDS.map<PcEnv>(tld => ({
+    pcEnvTld: tld,
+    pcAppOrigin: `https://apps.${tld}`
+}));
 
-    if (currEnvTld === DEFAULT_PC_ENV_TLD) {
-        DEFAULT_PC_ENV = currEnv;
-    }
-
-    result.push(currEnv);
-    return result;
-}, []);
+const [DEFAULT_PC_ENV] = PC_ENVS.filter(env => env.pcEnvTld === DEFAULT_PC_ENV_TLD);
```

Comment:
Did you use filter over find for browser compat issues?  Works fine, just curious.

Reply:
Oh no nothing to do with browser compact, just did this as I feel that it's cleaner since we don't need to do a non-null assertion, as we would with find since it returns `T | undefined`.